addons/crm/models/res_partner.py

```diff
@@ -77,19 +77,19 @@ def _compute_meeting(self):
             meeting_data = self.env.cr.fetchall()
 
             # Create a dict {partner_id: event_ids} and fill with events linked to the partner
-            meetings = {p.id: set() for p in all_partners}
+            meetings = {k: set() for k in set(m[0] for m in meeting_data)}
```

Comment:
Not sure why this needs to be `k` now, can't it be `p_id` so we know what it is without looking at the query?

Reply:
Sure!